resources/nftables.rb

```diff
@@ -40,7 +40,7 @@
 action :rebuild do
   ensure_default_rules_exist(new_resource)
 
-  file '/etc/nftables.conf' do
+  file nftables_conf_location do
```

Comment:
@bmhughes was asking for this to be based on the property and the default property value would then be based on this helper method. That way if the OS isn't supported yet users can manually specify the path through setting the property.

so this line would look something like `file new_resource.nftables_conf_path do` and then around line 32 you'd have the nftables_conf_path property defined.

Reply:
To add to this, an example of how the property definition should look [here](https://github.com/sous-chefs/nginx/blob/d1b9d7be4a4b4bc41c8283ce928b40073fd05a22/resources/config.rb#L71)